package/MDAnalysis/coordinates/DCD.py

```diff
@@ -514,7 +515,11 @@ def timeseries(self, asel, start=None, stop=None, step=None, format='afc'):
                where the shape is (frame, number of atoms,
                coordinates)
         """
-        start, stop, skip = self.check_slice_indices(start, stop, step)
+        if skip is not None:
+            step = skip
+        start, stop, step = self.check_slice_indices(start, stop, step)
+        # for old cython _read_timeseries code
+        skip = step
```

Comment:
Can't you get rid of this? Just use `step` throughout. We want people to use `step` because it behaves like `step`...